docs/UserGuide.md

```diff
@@ -266,47 +279,55 @@ Balance: -830.00
 Example output:
 
 ```
-You have 4 matching cashflows:
-1: Income
-   Type: Salary
-   Amount: 5000.00
-   Recurring every: 30 days, starting from: Oct 30 2023
-   Description: work
-2: Expense
+You have 6 matching cashflows:
+1: Expense
    Type: Dining
    Amount: 30.00
    Description: Genki Sushi
-3: Expense
-   Type: Insurance
-   Amount: 800.00
-   Recurring every: 365 days, starting from: Oct 30 2023
-   Description: ntuc income
-4: Expense
+2: Expense
    Type: Necessities
    Amount: 300.00
-   Recurring every: 30 days, starting from: Oct 30 2023
+   Recurring every: 30 days, date added: Nov 04 2023, recurring on: Dec 04 2023
    Description: groceries
+3: Income
+   Type: Allowance
+   Amount: 500.00
+   Recurring every: 30 days, date added: Nov 04 2023, recurring on: Dec 04 2023
+4: Income
+   Type: Investments
+   Amount: 1000.00
+5: Income
+   Type: Salary
+   Amount: 100.00
+6: Expense
+   Type: Others
+   Amount: 0.23
+Balance: 1269.77
```

Comment:
I think displaying one cashflow for each list is enough to prevent UG from getting too long